godot_project/editor/controls/dockers/workspace_docker/simulations_docker/simulations_docker.tscn

```diff
@@ -1,18 +1,8 @@
-[gd_scene load_steps=2 format=3 uid="uid://bystdhbca4tt8"]
```

Comment:
i am a little concerned by the fact the uid have changed, it may be being referenced somewhere else by uid. Can we change it back?